src/vector/_methods.py

```diff
@@ -1798,11 +1799,23 @@ class Vector2D(Vector, VectorProtocolPlanar):
     def to_Vector2D(self) -> VectorProtocolPlanar:
         return self
 
-    def to_Vector3D(self) -> VectorProtocolSpatial:
+    def to_Vector3D(self, **coord) -> VectorProtocolSpatial:
+        if len(coord) > 1:
+            raise ArgumentError(coord, "can be `z`, `theta`, or `eta`")
```

Comment:
This should not be an argparse error. It probably should be TypeError. But better yet, instead of manually parsing a `**`, it should list the three keywords = None then verify there are at least two None's.